grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy

```diff
@@ -46,6 +64,55 @@ class BioActivityController {
      * @param pActivityId project activity Id
      * @return
      */
+    @Operation(
+            method = "POST",
+            tags = "biocollect",
+            operationId = "bioactivityajaxupdate",
+            summary = "Create or edit an activity",
+            requestBody = @RequestBody(
+                    description = "JSON body",
+                    content = @Content(
+                            mediaType = "application/json",
+                            schema = @Schema(
+                                        implementation = ActivityAjaxUpdate.class
+                                )
+                    )
+            ),
+            parameters = [
+                    @Parameter(
+                            name = "id",
+                            in = ParameterIn.QUERY,
+                            description = "Activity id. Leave blank if creating new activity."
+                    ),
+                    @Parameter(
+                            name = "pActivityId",
+                            in = ParameterIn.QUERY,
+                            description = "Project activity id (survey id)"
+                    )
+            ],
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    schema = @Schema(
+                                            implementation = ActivitySaveResponse.class
+                                    )
+                            )
```

Comment:
Can you please add CORS headers section for all 200 responses?

https://confluence.csiro.au/display/ALASD/OIDC+Migration+on+ALA+Applications